lab_8/task_8_12.py

```diff
@@ -0,0 +1,17 @@
+import random
+
+
+def game():
+    while True:
+        text = input('Rock, paper or scissor? (print \'exit\' to end game) ').lower()
+        if text in ['scissor', 'rock', 'paper']:
+            result = random.choice(['Win', 'Draw', 'Lose'])
+            print(result)
```

Comment:
it's a joke?
so, you just randomly print 'Win', 'Draw', 'Lose'.
where is the game?